docs/SETUP.md

```
@@ -1,30 +1,27 @@
+### ✅ Step 4: Run the test suite
 ```
 uv run python -m unittest discover
 ```
 
-### 🚀 Step 4: Run the application
+### 🚀 Step 5: Run the application
 ```
 uv run flask run
 ```
 
 Open your favourite browser at http://127.0.0.1:5000/ and give it a try 🎉
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)